go.mod

```diff
@@ -14,12 +14,15 @@ require (
 	github.com/aws/aws-sdk-go v1.16.5
 	github.com/chrismcguire/gobberish v0.0.0-20150821175641-1d8adb509a0e
 	github.com/cpuguy83/go-md2man v1.0.8 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
```

Comment:
lots of little changes to `go.mod`, was `go mod tidy` run?

Reply:
Yes the build failed without running tidy.